omod/src/main/webapp/pages/visit/templates/sections/dxSectionHeading.gsp

```diff
@@ -12,6 +12,9 @@
     <span ng-repeat="diag in encounter.obs | byConcept:Concepts.diagnosisConstruct | withoutCodedMember:Concepts.diagnosisOrder:Concepts.primaryOrder">
         {{ diag | diagnosisShort }}{{ \$last ? "" : "," }}
     </span>
+    <span>
+        {{ ((encounter.obs | byConcept:Concepts.diagnosisConstruct | withCodedMember:Concepts.diagnosisOrder:Concepts.primaryOrder).length == 0 ) && (encounter.obs | byConcept:Concepts.diagnosisConstruct | withoutCodedMember:Concepts.diagnosisOrder:Concepts.primaryOrder).length == 0 ? "" : ", " }}{{ encounter.obs | byConcept:Concepts.nonCodedDiagnosis:true | obs:'value' }}
+    </span>
```

Comment:
can you explain the logic here, I was a little confused?

Reply:
Just puts a comma in front, if there are other coded diagnoses above:
![Screenshot 2024-10-28 at 4 56 11 PM](https://github.com/user-attachments/assets/81789eee-45cb-4791-9ed0-fb3fd7810951)

Reply:
LGTM @cioan

Reply:
Oh duh, I remember that now, thanks @cioan !